### ISSUE TITLE
Version on github does not match crates.io

### ISSUE THREAD

**Issue by ehuss** [rust-lang-nursery/rustfix#170]
The version on github is 0.4.4, but crates.io has 0.4.5 (published Mar 26, 2019).  I am a bit confused.

It also seems to be missing a change that is in the 0.4.5 version:

```diff
diff -r rustfix-0.4.5/src/diagnostics.rs /Users/eric/Proj/rust/rustfix/src/diagnostics.rs
4a5,6
> use serde::Deserialize;
>
diff -r rustfix-0.4.5/src/lib.rs /Users/eric/Proj/rust/rustfix/src/lib.rs
0a1,2
> #![warn(rust_2018_idioms)]
>
8,10c10
< #[macro_use]
< extern crate serde_derive;
< extern crate serde_json;
---
> use serde_json;
18c18
< use diagnostics::{Diagnostic, DiagnosticSpan};
---
> use crate::diagnostics::{Diagnostic, DiagnosticSpan};
47c47
<     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
---
>     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
59c59
<     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
---
>     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
181,182c181,182
<                     use Filter::*;
<                     use diagnostics::Applicability::*;
---
>                     use crate::Filter::*;
>                     use crate::diagnostics::Applicability::*;
```

**Comment by killercup** [comment]
Oh, that seems to be my fault! I just checked my local repo and it seems that I messed up my git config so it didn't have the correct URL for this repo set (it was moved from my account, and I later forked it).

The 0.4.5 tag was fa8a2f04f12b189854e8a4607d0786580be2dbd5 with the version changed. I've pushed it now but the precise commit it is unfortunately not on master. I'll also push the rebased version bump just to make this consistent again!
